--- check_mysql_perf-1.3.2.3/m4/np_officialplugins.m4~	2008-06-03 16:45:22.000000000 +0300
+++ check_mysql_perf-1.3.2.3/m4/np_officialplugins.m4	2009-02-16 10:31:45.573518818 +0200
@@ -26,16 +26,10 @@
 
   if test -z "$with_officialplugins"; then
     with_officialplugins="no"
-  else
-    if test -e $with_officialplugins/nagios-plugins.spec ; then
-      with_officialplugins=$with_officialplugins
-    else
-      with_officialplugins="no"
-    fi
   fi
   if test "x$with_officialplugins" != "xno" ; then
     np_official_plugins=$with_officialplugins
-    np_official_plugins_includes="-I${np_official_plugins} -I${np_official_plugins}/plugins -I${np_official_plugins}/gl -I${np_official_plugins}/lib"
+    np_official_plugins_includes="-I${np_official_plugins}/include/nagiosplug -I${np_official_plugins}/include/nagiosplug/plugins -I${np_official_plugins}/include/nagiosplug/gl -I${np_official_plugins}/include/nagiosplug/lib"
     np_official_plugins_libs="-L${np_official_plugins}/lib -lnagiosplug"
     np_official_plugins_cflags="$np_official_plugins_include $np_official_plugins_libs"
   fi
--- check_mysql_perf-1.3.2.3/plugins/Makefile.am~	2009-02-16 10:44:32.000000000 +0200
+++ check_mysql_perf-1.3.2.3/plugins/Makefile.am	2009-02-16 10:44:34.747432295 +0200
@@ -8,8 +8,8 @@
 DEFS = -DLOCALEDIR=\"$(localedir)\"
 
 libexec_PROGRAMS = check_mysql_perf
-BASEOBJS = $(OFFICIALPLUGINS)/plugins/utils.o $(OFFICIALPLUGINS)/lib/libnagiosplug.a $(OFFICIALPLUGINS)/gl/libgnu.a
-NETOBJS = $(OFFICIALPLUGINS)/plugins/netutils.o $(BASEOBJS) $(EXTRA_NETOBJS)
+BASEOBJS = $(OFFICIALPLUGINS)/lib/utils.o $(OFFICIALPLUGINS)/lib/libnagiosplug.a $(OFFICIALPLUGINS)/lib/libgnu.a
+NETOBJS = $(OFFICIALPLUGINS)/lib/netutils.o $(BASEOBJS) $(EXTRA_NETOBJS)
 
 test:
 	perl -I $(top_builddir) -I $(top_srcdir) ../test.pl
